### PR TITLE
Telegram notification changes

### PR DESCRIPTION
**Pull request by ice-myles**
New referral telegram scheduled notification with new referral information per last 24 hours. Socials telegram notification 5 mins after the user registration instead of 24 hours. One time telegram extend mining notification for inactive users spreaded among 4 hours. Warn about mining expiration telegram notification.

### CODE REVIEW

File: notifications/notification_type_socials.go [hunk 1]
```diff
@@ -45,6 +45,9 @@ func (r *repository) addScheduledSocialsNotifications(ctx context.Context, us *u
 			})
 		}
 	}
+	if !r.cfg.Development {
```
Comment (ice-ares) [inline notifications/notification_type_socials.go:48]: Huh?

File: notifications/notifications.go [hunk 2]
```diff
@@ -32,11 +32,15 @@ func New(ctx context.Context, _ context.CancelFunc) Repository {
 
 	ddlWorkersParam := fmt.Sprintf(ddl, schedulerWorkersCount)
 	db := storage.MustConnect(ctx, ddlWorkersParam, applicationYamlKey)
+	eskimoDB := storage.MustConnect(ctx, "", eskimoApplicationYamlKey)
```
Comment (ice-ares) [inline notifications/notifications.go]: U dont need this in this constructor

File: notifications/notification_type_socials.go (hunk 1 above)
Comment (ice-myles) [reply notifications/notification_type_socials.go:48]: It's the requirement to send from telegram in 5 minute. For pushes we didn't change anything. So for telegram development will be still 1 minute, for telegram non-development - 5 mins. For pushes everythins as-is.

File: notifications/notification_type_new_referral.go [hunk 3]
```diff
@@ -126,56 +127,65 @@ func (r *repository) sendNewReferralNotification(ctx context.Context, us *users.
...
 
+func (r *repository) getNewReferralCountLast24H(ctx context.Context, referredBy string) (info *newReferralCountLast24HInfo, err error) {
+	if ctx.Err() != nil {
+		return nil, errors.Wrap(ctx.Err(), "unexpected deadline")
+	}
+	sql := `SELECT 
+				COALESCE(count(*), 0) AS count,
+				(SELECT created_at FROM users WHERE id = $1) AS created_at
+			FROM users
+			WHERE referred_by = $1 AND created_at > $2`
```
Comment (ice-ares) [inline notifications/notification_type_new_referral.go:180]: do u have an index on it? Add it if u don't `(referred_by, created_at DESC)`

File: notifications/scheduler.go [hunk 4]
```diff
@@ -248,3 +255,88 @@ func (s *Scheduler) deleteScheduledNotifications(ctx context.Context, notificati
...
+	const limit = 10000
+	offset := uint64(0)
+	now := time.Now()
+	for {
+		sql := fmt.Sprintf(`SELECT 
+								id,
+								language
+							FROM users
+							WHERE last_mining_ended_at IS NOT NULL AND last_mining_ended_at < $1
+							ORDER BY created_at ASC
```
Comment (ice-ares) [inline notifications/scheduler.go:274]: create an index on `(last_mining_ended_at DESC NULLs LAST)`

**Pull request merged**